react/Checkbox/Checkbox.test.js

```diff
@@ -33,6 +33,18 @@ describe('Checkbox', () => {
     expect(wrapper).toMatchSnapshot();
   });
 
+  it('should render the invalid style for tone="critical" even if the field is valid', () => {
```

Comment:
To the same point, need a test case
```
  it('should render the invalid style for tone="someNonExistingTone" even if the field is valid', () => {
    expect(...)...;
  });```

Reply:
Are we testing anything else like this? If we were to write a test like that, what kind of value do you think it would provide us? We throw a prop type warning for an invalid value and I'm not sure we need to go any further than that